_config.yml

```diff
@@ -168,6 +168,12 @@ navigation:
      "*components/chart/tooltip":
         title: "Tooltips"
         position: 31
+     "*components/sankeydiagram/elements":
+        title: "Elements"
+        position: 5
+     "*components/sankeydiagram/customization":
+        title: "Customization"
```

Comment:
Customization is a too generic title, which has no practical value. Every usage of our components is subject to customization, and all the documentation of each component shows how to customize that component.

Why not move the articles from Customization to Elements and remove the Customization group altogether?